### ISSUE TITLE
Suspend unused indexers in Prod and compare metrics before/after

### ISSUE THREAD

**Issue by pkudinov** [near/queryapi#731]

**Comment by morgsmccauley** [comment]
I disabled 8 Indexer in Prod out of 152 total, so roughly 5%. These were the outcomes:
- Block Streamer:
    - BPS averages - unchanged
    - CPU - unchanged
    - Network data received - 64% reduction
    - Memory usage - 64% reduction
- Runner:
    - Execution duration averages - slight reduction, probably related to these Indexers not being executed
    - Heap allocation - 30% reduction
    - Network data received - 69% reduction
    - Memory usage - 15% reduction
    
    
To summarise, there was a significant impact on system performance, but Indexer performance was mostly unchanged. I believe this would be helpful if we start hitting the upper limit of our resources, but scaling up could also help.
